src/components/NavBar.jsx

```diff
@@ -0,0 +1,81 @@
+"use client";
+import React, { useState } from "react";
+import Image from "next/image";
+import Link from "next/link";
+import logo from "@/public/logo.png";
```

Comment:
do not import images like this, when they are in public they can be referenced such as `/logo.png`
for more information: https://nextjs.org/docs/app/building-your-application/optimizing/static-assets